### PR TITLE
Plugins: Add faceDetect

### PR DESCRIPTION
**Pull request by MinjeAhn**
Add face detect plugin and update README.md

### CODE REVIEW

**Review by jiyong2 — APPROVED** [review]
Thank you for adding a new plugin. Approved.

File: plugins/faceDetect/faceDetect.cpp [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+		throw ovi::Exception(OVI_ERROR_INVALID_OPERATION, "not supported format");
+	}
+
+	if (!image.data)
+		throw ovi::Exception(OVI_ERROR_INVALID_OPERATION, "image.data is empty");
+
+	if (_faceCascade.empty()) {
+		_faceCascade.load(HAAR_FACE);
+		if (_faceCascade.empty())
+			throw ovi::Exception(OVI_ERROR_INVALID_OPERATION, "cascade load failed.");
```
Comment (haejeongkim) [inline plugins/faceDetect/faceDetect.cpp]: nit: how about "face cascade load failed" to distinguish it from eye cascade?

File: plugins/faceDetect/faceDetect.cpp [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+		_eyeCascade.load(HAAR_EYES);
+		if (_eyeCascade.empty())
+			throw ovi::Exception(OVI_ERROR_INVALID_OPERATION, "cascade load failed.");
```
Comment (haejeongkim) [inline plugins/faceDetect/faceDetect.cpp]: nit: same as above.

File: plugins/faceDetect/faceDetect.cpp (hunk 1 above)
Comment (MinjeAhn) [reply plugins/faceDetect/faceDetect.cpp]: Done.

File: plugins/faceDetect/faceDetect.cpp (hunk 2 above)
Comment (MinjeAhn) [reply plugins/faceDetect/faceDetect.cpp]: Done.

**Review by haejeongkim — APPROVED** [review]

**Pull request merged**